### PR TITLE
Contrib/CDS Google Chronicle Backstory release 2.1.0

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
* Implemented 14 new commands
* Updated the default value for "*How many incident to fetch each time*" to '**100**'.
* Added tooltip for "*first fetch time*".
* Updated the field label for "*Select the severity of asset alerts to be filtered for Fetch Incidents. Available options are 'High', 'Medium', 'Low' and 'Unspecified' (Default-No Selection).*" to "*Select the severity of alerts to be filtered for Fetch Incidents. Available options are 'High', 'Medium', 'Low' and 'Unspecified' (If not selected, fetches all alerts).*".
* Updated error messages for arguments of date type.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by MosheEichler — CHANGES_REQUESTED** [review]
HI [crestdatasystems](https://github.com/crestdatasystems)
You did an amaizing job
Please see my few comments.

File: Tests/conf.json [hunk 1]
```diff
@@ -3157,6 +3157,21 @@
...
+        {
+            "integrations": "Google Chronicle Backstory",
+            "playbookID": "playbook-GoogleChronicleBackstoryRules-Test",
+            "fromversion": "5.0.0"
+        },
+        {
+            "integrations": "Google Chronicle Backstory",
+            "playbookID": "playbook-GoogleChronicleBackstoryReferenceList-Test",
+            "fromversion": "5.0.0"
+        },
```
Comment (ShahafBenYakir) [inline Tests/conf.json:3174]: No need to add these to conf.json, we don't run partner test PB as part of our internal build anymore.

File: Packs/GoogleChronicleBackstory/.pack-ignore [hunk 2]
```diff
@@ -63,4 +63,7 @@ ignore=PB114
 ignore=IM111
 
 [file:classifier-Chronicle.json]
-ignore=BA101
\ No newline at end of file
+ignore=BA101
+
+[file:conf.json]
+ignore=BA102
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/.pack-ignore]: Yes, Removed this from the .pack-ignore file.

File: Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py [hunk 3]
```diff
@@ -1198,38 +1240,22 @@ def validate_and_parse_detection_start_end_time(args: Dict[str, Any]) -> Tuple[s
     :param args: contains all arguments for command
 
     :return : detection_start_time, detection_end_time: Detection start and end time in the format API accepts
-    :rtype : Tuple[str, str]
+    :rtype : Tuple[Optional[datetime], Optional[datetime]]
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py]: Yes, Updated.

File: Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py [hunk 4]
```diff
@@ -2236,194 +2266,938 @@ def get_list_rules_hr(rules: List[Dict[str, Any]]) -> str:
...
+    :param client_obj: client object which is used to get response from api
+
+    :type params: Dict[str, Any]
+    :param params: it contain configuration parameter
+
+    :return: raise ValueError if any error occurred during connection
+    :rtype: None
+    """
+    demisto.debug('Running Test having Proxy {}'.format(params.get('proxy')))
+    request_url = '{}/ioc/listiocs?start_time=2019-10-15T20:37:00Z&page_size=1'.format(
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py:3031]: Yes, it's working and we're just checking the connection with API.

File: Tests/conf.json (hunk 1 above)
Comment (crestdatasystems) [reply Tests/conf.json]: Ok, Removed these entries from conf.json.

**Review by MosheEichler — APPROVED** [review]

**Pull request merged**